app/services/plans/create_service.rb

```diff
@@ -33,6 +33,14 @@ def create(args)
           taxes_result.raise_if_error!
         end
 
+        if args[:usage_thresholds].present? &&
+            License.premium? &&
```

Comment:
How do you feel about something like: `License.supports_progressive_billing?(plan.organization)`

Reply:
@nudded That's a good point which we already talked about a couple of time in the past.

We discussed using feature flags (e.g. [LaunchDarkly](https://launchdarkly.com/feature-flags-ruby/)) for this purpose with @jdenquin but since it was not decided yet, we can keep it like this.
That might affect implementation of `License` as well. 

Any opinions @jdenquin @lovrocolic @vincent-pochet